### ISSUE TITLE
hover a node behaviour

### ISSUE THREAD

**Issue by natael** [varusgarcia/Design-History-InfoVis#19]
lower opacity for unrelevant nodes.

**Comment by alexkaessner** [comment]
@natael  I've converted the `test.html` implementation to `app.js`.
As we use group elements (g) for the nodes, I can't manage to connect the lines to the selected group. Do you have any idea?

I'm currently using this code in `app.js` (line 265+):

```
nodeBlock.on("mouseover", function(d){
      node.select("circle").classed("node-active", function(o) {
          thisOpacity = isConnected(d, o) ? true : false;
          this.setAttribute('fill-opacity', thisOpacity);
          return thisOpacity;
      });

      link.classed("link-active", function(o) {
          return o.source === d || o.target === d ? true : false;
      });

      d3.select(this).classed("node-active", true);
      d3.select(this).select("circle").transition()
              .duration(50)
              .attr("r", (d.weight * 2 + baseCircleRadius)*1.2);
  });

nodeBlock.on("mouseout", function(d){
      node.classed("node-active", false);
      link.classed("link-active", false);

      d3.select(this).select("circle").transition()
              .duration(200)
              .attr("r", d.weight * 2 + baseCircleRadius);
  });
```

**Comment by vale2k7** [comment]
taken out from: http://jsfiddle.net/2pdxz/2/

nodes.on('mouseover', function(d) {
  link.style('stroke-width', function(l) {
    if (d === l.source || d === l.target)
      return 4;
    else
      return 2;
    });
});

// Set the stroke width back to normal when mouse leaves the node.
nodes.on('mouseout', function() {
  link.style('stroke-width', 2);
});

**Comment by alexkaessner** [comment]
Relating: http://stackoverflow.com/questions/8739072/highlight-selected-node-its-links-and-its-children-in-a-d3-force-directed-grap

Relating 2: http://bl.ocks.org/emeeks/9357371

**Comment by vale2k7** [comment]
What about this:

http://stackoverflow.com/questions/19111581/d3js-force-directed-on-hover-to-node-highlight-colourup-linked-nodes-and-link

**Comment by alexkaessner** [comment]
fixed by @vale2k7 ab86b4e
